src/components/Users/User.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import userPhoto from "../../assets/images/user.png";
+import { NavLink } from "react-router-dom";
+import Button from "@material-ui/core/Button";
+import styled from "styled-components";
+
+const User = ({ users, followingInProgress, unfollow, follow }) => {
+  return (
+    <div>
+      {users.map((user) => (
+        <StyledUserItems key={user.id}>
+          <span>
+            <div>
+              <NavLink to={`/profile/${user.id}`}>
+                <StyledAvatar src={user?.photos?.small || userPhoto} />
+              </NavLink>
+            </div>
+            <div>
+              {user.followed ? (
+                <Button
+                  size="small"
+                  variant="contained"
+                  color="primary"
+                  disabled={followingInProgress.some((id) => id === user.id)}
+                  onClick={() => {
+                    unfollow(user.id);
+                  }}
+                >
+                  unFollow
+                </Button>
+              ) : (
+                <Button
+                  size="small"
+                  variant="contained"
+                  color="primary"
+                  disabled={followingInProgress.some((id) => id === user.id)}
+                  onClick={() => {
+                    follow(user.id);
+                  }}
+                >
+                  Follow
+                </Button>
+              )}
+            </div>
+          </span>
+          <span>
+            <div>{user.name}</div>
+            <div>{user.status}</div>
+          </span>
+        </StyledUserItems>
```

Comment:
Bad naming - in the future you can has 100500 components like "StyledText", "StyledContainer", "StyledUser", "Styled....". Please, set the name more semantically like this - UsersContainer, HeaderTtiles, AvatarContainer, etc. All of them are styled of course.